promql/engine.go

```diff
@@ -1350,7 +1350,7 @@ func (ev *evaluator) matrixIterSlice(it *storage.BufferedSeriesIterator, mint, m
 			ev.currentSamples++
 		}
 	}
-	// The seeked sample might also be in the range.
+	// The sought sample might also be in the range.
```

Comment:
Seeked is a non-standard, but correct past tense of seek, and in these cases easier to read than changing the word to sought.

Reply:
OK Thank You

Reply:
I Changed it Any problem?